### PR TITLE
Code improvements

### PR DESCRIPTION
**Pull request by sebm253**
This PR _improves_ some parts of the code.

### CODE REVIEW

**Comment by dscalzi** [comment]
I see you loaded this up in intellij lol

File: ZipExtractor-Bukkit/src/main/java/com/dscalzi/zipextractor/bukkit/ZipExtractor.java [hunk 1]
```diff
@@ -18,25 +18,19 @@
...
 import com.dscalzi.zipextractor.core.util.IPlugin;
+import org.bukkit.plugin.java.JavaPlugin;
 
-public class ZipExtractor extends JavaPlugin implements IPlugin {
+import java.util.ArrayList;
+import java.util.List;
+import java.util.logging.Level;
 
-    @SuppressWarnings("unused")
-    private Metrics metrics;
```
Comment (dscalzi) [inline ZipExtractor-Bukkit/src/main/java/com/dscalzi/zipextractor/bukkit/ZipExtractor.java]: Don't remove this.

File: ZipExtractor-Bukkit/src/main/java/com/dscalzi/zipextractor/bukkit/ZipExtractor.java [hunk 2]
```diff
@@ -45,7 +39,6 @@ public void onEnable() {
         ZServicer.initalize(ConfigManager.getInstance().getMaxQueueSize(),
                 ConfigManager.getInstance().getMaxPoolSize());
         this.getCommand("zipextractor").setExecutor(new MainExecutor(this));
-        metrics = new Metrics(this);
```
Comment (dscalzi) [inline ZipExtractor-Bukkit/src/main/java/com/dscalzi/zipextractor/bukkit/ZipExtractor.java]: Don't remove this.

**Review by dscalzi — CHANGES_REQUESTED** [review]

File: ZipExtractor-Bukkit/src/main/java/com/dscalzi/zipextractor/bukkit/ZipExtractor.java (hunk 2 above)
Comment (sebm253) [reply ZipExtractor-Bukkit/src/main/java/com/dscalzi/zipextractor/bukkit/ZipExtractor.java]: Why? `metrics` variable was not used at all, you even suppressed the warning of it not being used in the comment above.

Comment (dscalzi) [reply ZipExtractor-Bukkit/src/main/java/com/dscalzi/zipextractor/bukkit/ZipExtractor.java]: It's to turn on metrics with bStats. Just how their API works.

Comment (sebm253) [reply ZipExtractor-Bukkit/src/main/java/com/dscalzi/zipextractor/bukkit/ZipExtractor.java]: I think the variable is not needed at all, i think doing `new Metrics(this)` would be enough.

Comment (dscalzi) [reply ZipExtractor-Bukkit/src/main/java/com/dscalzi/zipextractor/bukkit/ZipExtractor.java]: I'd rather have the variable there

Comment (sebm253) [reply ZipExtractor-Bukkit/src/main/java/com/dscalzi/zipextractor/bukkit/ZipExtractor.java]: Re-added.

**Review by dscalzi — APPROVED** [review]

**Pull request merged**